src/main/kotlin/no/nav/amt/arena/acl/processors/DeltakerProcessor.kt

```diff
@@ -43,7 +43,7 @@ open class DeltakerProcessor(
 		val arenaDeltaker: TiltakDeltaker = data.getMainObject<ArenaTiltakDeltaker>().mapTiltakDeltaker()
 
 		val gjennomforingInfo =
-			idTranslationRepository.get(TILTAKGJENNOMFORING_TABLE_NAME, arenaDeltaker.tiltakgjennomforingId)
+			arenaDataIdTranslationService.findGjennomforingIdTranslation(arenaDeltaker.tiltakgjennomforingId)
```

Comment:
Denne kunne evt kastet exception selv om den ikke brukes til noe annet. og ville brukt `get` isteden for `find` i navngivning

Reply:
Kan godt bytte til `get`, men ønsker ikke at den skal kaste et exception.